src/pool/srv_pool.c

```diff
@@ -925,14 +928,52 @@ handle_event(struct pool_svc *svc, struct pool_svc_event *event)
 		goto out_prop;
 	}
 
-	rc = pool_svc_exclude_rank(svc, event->psv_rank);
-	if (rc != 0) {
-		D_ERROR(DF_UUID": failed to exclude rank %u: "DF_RC"\n", DP_UUID(svc->ps_uuid),
-			event->psv_rank, DP_RC(rc));
-		goto out_prop;
-	}
+	if (DAOS_POOL_SUPPORT_REINT_ONLINE == 0)
+		goto DEAD;	
 
-	D_DEBUG(DB_MD, DF_UUID": excluded rank %u\n", DP_UUID(svc->ps_uuid), event->psv_rank);
+	if (event->psv_type == CRT_EVT_ALIVE) {
+        dom = pool_map_find_node_by_rank(svc->ps_pool->sp_map, event->psv_rank);
+		if (dom == NULL) {
+			D_ERROR("rank %u not in map \n",event->psv_rank);
+			goto out_prop;
+		}
```

Comment:
many places in this patch is not well aligned, and some places was using space rather than hard tab.
(If you are using VIM, may set "set tabstop=8" in your vimrc, just FYI).